packages/rollup-plugin-html/README.md

```diff
@@ -452,19 +505,31 @@ All configuration options are optional if an option is not set the plugin will f
 
 Type: `string`
 
-Name of the generated HTML file. If `inputPath` is set, defaults to the `inputPath` filename, otherwise defaults to `index.html`.
+Name of the generated HTML file. If `files` is set, defaults to the `files` filename, otherwise defaults to `index.html`.
 
-### inputPath
+### files
 
-Type: `string`
+Type: `string|strings[]`
 
-Path to the HTML file to use as input. Modules in this file are bundled and the HTML is used as the template for the generated HTML file.
+Paths to the HTML file to use as input. Modules in this files are bundled and the HTML is used as the template for the generated HTML file. Does support glob patterns.
 
-### inputHtml
+### rootDir
 
 Type: `string`
 
-Same as `inputPath`, but provides the HTML as a string directly.
+Path where the `files` should start look.
+
+### flatten
+
+Type: `boolean`
+
+Should folder in filePaths be stripped and all files be places in a single folder (Default to true)
+
+### html
+
+Type: `string|[{ name: string, html: string, rootDir: string}]`
```

Comment:
I think we can remove `rootDir` here now, and use the top level option.

Reply:
this "rootDir" is the one from the html file 😅
and is different then the "global" rootDir... probably a good idea to rename it..

pageRootDir and htmlRootDir or rootDir?

Reply:
We can probably reuse the same variable for both

Reply:
ok will keep the same name 👍